bundles/org.openhab.binding.tacmi/src/main/java/org/openhab/binding/tacmi/internal/json/obj/Data.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.tacmi.internal.json.obj;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Class holding the Data JSON element
+ *
+ * @author Moritz 'Morty' Strübe - Initial contribution
+ */
+@NonNullByDefault
+public class Data {
+
+    public Collection<IO> inputs;
+    public Collection<IO> outputs;
+    public Collection<IO> general;
+
+    public Data() {
+        inputs = Collections.emptyList();
+        outputs = Collections.emptyList();
+        general = Collections.emptyList();
+    }
```

Comment:
You could avoid the constructor and set directly each member like for example:
`public Collection<IO> inputs = Collections.emptyList();`